packages/i18n/src/locales/en.i18n.json

```diff
@@ -4958,7 +4958,7 @@
   "SMS": "SMS",
   "SMS_Description": "Enable and configure SMS gateways on your workspace.",
   "SMS_Default_Omnichannel_Department": "Omnichannel Department (Default)",
-  "SMS_Default_Omnichannel_Department_Description": "If set, all new incoming chats initiated by this integration will be routed to this department.  \nThis setting can be overwritten by passing department query param in the request.  \ne.g. `https://{{SERVER_URL}}/api/v1/livechat/sms-incoming/twilio?department={{Department Id or Name}}`.  \nNote: if you're using Department Name, then it should be URL safe.",
```

Comment:
Please remove the unrelated changes from these translation strings.

Reply:
Done!